deploy/amp/database_initialise.php

```diff
@@ -8,9 +8,9 @@
 // james: uncomment the following lines to also initialise the
 //        primary 'buttonmen' database
 //
-//        recreate_database('buttonmen');
-//        $conn = db_connect('buttonmen');
-//        run_all_sql_files($conn);
+        recreate_database('buttonmen');
+        $conn = db_connect('buttonmen');
+        run_all_sql_files($conn);
```

Comment:
Did you intend to commit this change?  I don't really care about changes to this file, but it's a little distracting --- maybe we should talk about how you invoke this file, and add a CLI flag you can use to also recreate the primary database, so that you don't need to do that by modifying the file.

(FWIW, what /usr/local/bin/create_buttonmen_databases does is always drop-and-recreate buttonmen_test, and recreate buttonmen only if it does not exist.  This is a safety feature (and, ahem, it did save my bacon on one of the real sites once), but it means that my test process can be to drop the buttonmen database and then just run the script as usual.)

Anyway, for purposes of this pull, i just need to know whether you want this change in here, or want to remove it.  But the previous paragraph might have some thoughts for the future.